project-addons/flask_middleware_connector/events/picking_events.py

```diff
@@ -88,7 +88,8 @@ def delay_export_picking_write(session, model_name, record_id, vals):
             and picking.picking_type_id.code == 'outgoing' \
             and not picking.not_sync \
             and picking.company_id.id == 1:
-        if 'name' in vals or 'partner_id' in vals:
+        if 'name' in vals or 'partner_id' in vals or \
+                ('not_sync' in vals and not vals['not_sync']):
```

Comment:
¿esto es correcto? porque en la descripción indicas que quieres que se genere cuando pase de no sincronizable a sincronizable y aquí parece que haces lo contrario, aunque puede ser una confusión por el nombre del campo.

Reply:
Creo que es correcto, porque al desmarcar el check not_sync (que significaría que pasa a ser sincronizable), en el vals vendría como False, por lo que _"not False" = True_ y se cumpliría la condición.

Reply:
Además está probado y funciona como debería (al menos para los casos de prueba)

Reply:
OK